### ISSUE TITLE
Replace PSU

### ISSUE THREAD

**Issue by huffeec** [DoESLiverpool/circular_knitic#7]
We have a neater RS power supply that can replace the old computer unit.
It needs wiring and an enclosure that can be fixed under the box of electronics.
(Or the whole box of electronics redesigning?)

**Comment by amcewen** [comment]
We've decided against the RS power supply now, haven't we?  Wasn't @paulgeering proposing switching to a beefy 12V wall-wart, so the barrel-jack plugs into the Arduino and we take the stepper feed from the Vin pin of the Arduino?

If so, how much current do we think the wall-wart needs to provide?

**Comment by paulgeering** [comment]
Well, in the run up to Newcastle, I got a 12v 5A (60W) power supply for it and hooked it all up.
It was felt, by some experienced knitic users, that there was a noticeable drop in performance when using the new power supply, so we went back to using the PC power supply.

Shame the knitic is in pieces, otherwise we could run it and measure the current draw at 12v.

I guess we could put an artificial load on the stepper, and see what it maxes out at.
I'm around this evening, and happy to give that a go.

@amcewen do you have specs of the motor driver board max current draw, or is it the case that there is a recommended max, but the little board will try and keep up regarless of whether it is melting itself in the process?

**Comment by amcewen** [comment]
@paulgeering It's a [Big Easy Driver board](http://www.schmalzhaus.com/BigEasyDriver/) which (according to that link) will handle around 2A per phase.  We could always try it on the bench power supply and look at how much current it draws in operation (with some wool in it), and then get something specced a healthy margin over that...

**Comment by paulgeering** [comment]
On Saturday we measured the current draw at under 2 amps, we then did an A-B comparison between the PC power supply and the maplin one I mentioned.

They performed identically.

I've wired the knitic up so it will use the new power supply. 

(The knitic is awaiting a replacement Arduino. I blew the current one up, as is my custom in the lead up to Maker fair/fest's.

There is a separate git hub issue for that, which will be resolved on Monday morning)
